app/Resources/views/committee/members.html.twig

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+    Membres du comité « {{ committee.name }} » {{ parent() }}
+{% endblock %}
+
+{% block banner '' %}
+
+{% block content %}
+    {% set is_host = is_host(committee) %}
+
+    <section class="l__wrapper--medium">
+        <h1>{{ committee.name }}</h1>
+
+        <a href="#">Discussion</a>
+        <strong>Membres</strong>
+
+        <a href="{{ path('app_committee_edit', { uuid: committee.uuid, slug: committee.slug } ) }}">Editer le comité</a>
```

Comment:
Use `{{ committee_path('app_committee_edit', committee) }}` instead and make sure this link is visible only for committee hosts.